scripts/prepare-ci-environment.sh

```diff
@@ -10,8 +10,10 @@ fi
 # Python plugin for Community and Ultimate Edition
 
 export CWD=$(pwd)
-export IJ_VERSION="191.6183.62"
-export IJ_BUILD_NUMBER="191.6183.62"
+# Normally, IJ_VERSION is of the form YEAR.x[.y[.z]]
+# But for EAPs, set IJ_VERSION to the same as IJ_BUILD_NUMBER
```

Comment:
nit: we can remove these comments as this is no longer for EAP version.

Reply:
It is still needed actually to remind people how this works, e.g. when 2019.3 comes around.